news/feature-3245.md

```diff
@@ -0,0 +1,13 @@
+`example-msg-generator`: support fast message generation
```

Comment:
This snippet will be added to the news file. We need something different than the original issue text.

```
`example-msg-generator`: support `freq(0)` for fast message generation

log {
   source { example-msg-generator(freq(0) num(100)); };
   destination { file("/dev/stdout"); };
};
```
is enough. The caution need to be added only to the documentation, if it is true at all. I have a little doubts now. I just tested your code, and did not run into trouble without flow control, so I may have been wrong about it.

Reply:
@furiel : okay, thanks for the review